pyCGM_Single/pycgmStatic.py

```diff
@@ -1698,43 +2036,47 @@ def rotaxis_nonfootflat(frame,ankle_JC):
     return [R,L,foot_axis]
  
 def getankleangle(axisP,axisD):
-    """
-    
-    Static angle calculation function.
+    """Static angle calculation function.
     
     This function takes in two axis and returns three angles.
     and It use inverse Euler rotation matrix in YXZ order.
     the output shows the angle in degrees.
     
     As we use arc sin we have to care about if the angle is in area between -pi/2 to pi/2
     but in case of calculate static offset angle it is in boundry under pi/2, it doesn't matter.
-    ------------------------------------------------------
     
-    INPUT: each axis show the unit vector of axis.
-            axisP = [[axisP-x axis x,y,z position],
+    Parameters
+    ----------
+    axisP : array
+        Shows the unit vector of axisP.
+           axisP = [[axisP-x axis x,y,z position],
```

Comment:
This style seems to deviate from what has been done previously

Reply:
I'm a bit confused, do you want me to remove the code layout for the parameter? Or make it so that there is one multi-parameter description like `axisP, axisD : array` since they are very similar?

Reply:
Resolved the style for several of the relevant functions.